solutions/quickstart/DA_docs.md

```diff
@@ -0,0 +1,117 @@
+# Configuring complex inputs for OpenShift Virtualization in IBM Cloud projects
+
+Several optional input variables in the Red Hat OpenShift cluster [Deployable Architecture](https://cloud.ibm.com/catalog#deployable_architecture) use complex object types. You can specify these inputs when you configure your deployable architecture.
```

Comment:
Just to confirm - which DA we are talking about here.
`input variables in the Red Hat OpenShift cluster [Deployable Architecture]` - is it correct?